deployability/modules/testing/tests/test_agent/test_uninstall.py

```diff
@@ -79,8 +79,8 @@ def test_uninstall(wazuh_params):
 
     # Agent uninstallation
     for agent_names, agent_params in wazuh_params['agents'].items():
-        WazuhAgent.perform_uninstall_and_scan_for_agent(agent_params,wazuh_params)
-
+        #WazuhAgent.perform_uninstall_and_scan_for_agent(agent_params,wazuh_params)
```

Comment:
Remove comment

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5324/commits/96c47257f1bca33c7f144285a4c7b63a8c10fbce